<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestore Localizzazione</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Cerca</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inp_search"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Traduci da</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_from">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>a:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_to">
        <item>
         <property name="text">
          <string>it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_translate">
        <property name="text">
         <string>Traduci</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="lsw_strings">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="inp_string_content"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_open_arb"/>
    <addaction name="act_save_arb"/>
    <addaction name="act_save_arb_as"/>
   </widget>
   <widget class="QMenu" name="mnu_modifica">
    <property name="title">
     <string>Modifica</string>
    </property>
    <addaction name="act_new_string"/>
    <addaction name="act_delete_string"/>
    <addaction name="act_rename_string"/>
   </widget>
   <addaction name="mnu_file"/>
   <addaction name="mnu_modifica"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_open_arb">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Apri File ARB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_save_arb">
   <property name="text">
    <string>Salva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_save_arb_as">
   <property name="text">
    <string>Salva con nome...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="act_new_string">
   <property name="text">
    <string>Nuova Stringa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_delete_string">
   <property name="text">
    <string>Elimina Stringa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="act_rename_string">
   <property name="text">
    <string>Rinomina Stringa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
